### PR TITLE
[RF] POM | AT_17.04_001 | Verify that user can move org folder through Jenkins Dashboard

### PR DESCRIPTION
**Pull request by EkBlinova**
https://trello.com/c/GiGkrC1w/1815-rf-at1704001-verify-that-user-can-move-the-organization-folder-through-the-jenkins-dashboard

### CODE REVIEW

File: cypress/pageObjects/HomePage.js [hunk 1]
```diff
@@ -429,6 +430,11 @@ class HomePage {
         this.getDeleteOrgFolderDrpDwnMenuBtn().click();
         return new OrgFolderDeletePage();
     }
+
+    hoverAndClickOrgFolderDrpDwnMenuBtn() {
+        this.getOrgFolderDrpDwnMenuBtn().realHover().click()
+        return this
+    }
```
Comment (Rom-T) [inline cypress/pageObjects/HomePage.js:437]: Why create a separate method? It was possible to use the hoverAndClickProjectDrpDwnBtn method and pass the project name there.

Comment (EkBlinova) [reply cypress/pageObjects/HomePage.js:437]: How to pass the project name there?

Comment (DimaCurkanu) [reply cypress/pageObjects/HomePage.js:437]: Katia! Try something like that:   .hoverAndClickProjectDrpDwnBtn(folderName)  in organizationFolder.cy.js

Comment (EkBlinova) [reply cypress/pageObjects/HomePage.js:437]: Thanks Dima.
I tried to use the .hoverAndClickProjectDrpDwnBtn(newItemPageData.orgFolderName)
and it shows me that I have to use {multiple: true}, should I add  {multiple: true} to the .hoverAndClickProjectDrpDwnBtn(newItemPageData.orgFolderName) method?

image

Comment (Rom-T) [reply cypress/pageObjects/HomePage.js:437]: Yes that's right. Please fix the hoverAndClickProjectDrpDwnBtn method:
```
hoverAndClickProjectDrpDwnBtn(projectName) {
        this.getProjectNameLink().contains(projectName).realHover();
        this.getProjectDrpDwn(projectName).click();
        return this;
    }
```

Comment (DimaCurkanu) [reply cypress/pageObjects/HomePage.js:437]: I guess - no. It shows you that your method finds 2 elements instead 1. I think you need more exact locator or check methods you used again

Comment (Rom-T) [reply cypress/pageObjects/HomePage.js:437]: That's what I wrote. Changed it a little

File: cypress/pageObjects/HomePage.js [hunk 2]
```diff
@@ -118,6 +118,7 @@ class HomePage {
     getDashboardElement = () => cy.get('.dashboard');
     getDeleteOrgFolderDrpDwnMenuBtn = () =>
         cy.get("#breadcrumb-menu li:nth-child(4) span")
+    getOrgFolderDrpDwnMenuBtn = () => cy.get('a[href="job/OrganizationFolderJob/"] button')
```
Comment (Rom-T) [inline cypress/pageObjects/HomePage.js:121]: Please delete this element

**Review by DimaCurkanu — APPROVED** [review]

**Pull request merged**